src/ObjectPool/ObjectPool.sln

```diff
@@ -0,0 +1,42 @@
+﻿
```

Comment:
Remove/replace. We use slnf in this repository. e.g. https://github.com/dotnet/aspnetcore/blob/main/src/Http/HttpAbstractions.slnf

Reply:
Whoops, that slipped in...